README.md

```diff
@@ -43,7 +43,7 @@ Spell Revisions latest release or prerelease, if there is no release or prerelea
 http://lynxlynx.info/ie/modhub.php?Gibberlings3/SpellRevisions&ifeellucky
 
 Spell Revisions universal zip package from latest release:
-http://lynxlynx.info/ie/modhub.php?Gibberlings3/SpellRevisions&pkg=uzp
+http://lynxlynx.info/ie/modhub.php?InfinityMods/ModPackage&pkg=uzp
```

Comment:
This returns an html doc for me, not a zip. Plus the explanation of the url still mentions spell revisions.

Reply:
Fixed.
The 'win' works: https://lynxlynx.info/ie/modhub.php?InfinityMods/ModPackage&pkg=win but the uzp code is not yet committed so it doesn't work yet.